data.sql

```diff
@@ -79,4 +79,71 @@ UPDATE animals
 SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
 WHERE name = 'Angemon' OR name = 'Boarmon';
 
+-- Insert the following data for vets:
+-- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+-- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+-- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+-- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES
+('Vet William Tatcher', 45, '2000-04-23'),
+('Vet Maisy Smith', 26, '2019-01-17'),
+('Vet Stephanie Mendez', 64, '1981-05-04'),
+('Vet Jack Harkness', 38, '2008-06-08');
+
+-- Insert the following data for specializations:
+-- Vet William Tatcher is specialized in Pokemon.
+-- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
+-- Vet Jack Harkness is specialized in Digimon.
+
+INSERT INTO specializations (vet_id, species_id) VALUES
+(1, 1),
+(3, 2),
+(4, 2);
```

Comment:
- [ ] `Vet Stephanie Mendez` should have two animals. Pokemon and Digimon. By the look of things you only insert for Digimon. Please insert for Pokemon.

Example:
` (3, 1)`